src/xunit.performance.api/ScenarioTest.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace Microsoft.Xunit.Performance.Api
 {
-    public sealed class Scenario : IDisposable
+    public sealed class ScenarioTest : IDisposable
     {
-        public Scenario(ScenarioConfiguration configuration)
+        public ScenarioTest(ScenarioTestConfiguration configuration)
```

Comment:
@dsplaisted, can you share some context on the name change from Scenario to ScenarioTest?  I'm looking for a concrete example of the overall goal of this change.  It sounds to me, but I've not been able to confirm this in code, that the goal is to have scenario, like "Build", but have it run multiple flavors of that scenario with very minimal changes - clean, no changes.  Thus, a scenario is comprised of multiple tests, each which are slightly different.  Assuming I'm understanding things correctly here, do we need to make this change to the overall architectural model, or can we have multiple scenario benchmarks that simply run very similar code because they are similar.  I think I might be missing the reason why the existing model isn't sufficient to meet this goal.

Reply:
I should also say that if we do need this, I'm OK with extending the model - I just want to understand what's missing from the current model.

Reply:
@brianrob I was matching the structure of the XML result file that's generated.  The ScenarioBenchmark is the root element, and underneath it there can be multiple tests (and each test can have multiple iterations).

It seemed useful to me to be able to group the tests instead of having a flat list.  In this case, the organization I went with was to have the top level groups (scenarios) be a given project, and then the tests were different operations that would be performed on that project (such as clean build, incremental build, restore, etc).

Reply:
Thanks @dsplaisted.  Does that mean that the way you run the benchmarks is unaffected - the change just affects the organization of test results?